enterprise_subsidy/apps/api/v1/tests/test_views.py

```diff
@@ -374,18 +374,12 @@ def test_valid_delete_subsidy_by_role(self, role, status_code):
             self.set_up_operator()
         response = self.client.delete(self.get_details_url([self.subsidy_1.uuid]))
 
+        subsidy = Subsidy.all_objects.get(uuid=self.subsidy_1.uuid)
         self.assertEqual(response.status_code, status_code)
-
-    def test_delete_subsidy_with_invalid_uuid(self):
-        self.set_up_operator()
-        response = self.client.delete(self.get_details_url([str(uuid.uuid4())]))
-
-        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
-        self.assertDictEqual(response.json(), {
-            "error_code": "Forbidden",
-            "developer_message": "MISSING: subsidy.can_write_subsidies",
-            "user_message": "MISSING: subsidy.can_write_subsidies",
-        })
```

Comment:
Is there a reason for deleting this test?

Reply:
You're right, it still applies, I'll add it back in